.github/workflows/release-community-operator-PRs.yml

```diff
@@ -7,6 +7,9 @@ on:
       forceflag:
         description: 'To update an existing PR, use -f or --force flag here'
         default: ''
+      platforms:
+        description: 'List platforms for which to create PRs, which could be \'openshift\', \'kubernetes\' or both
+        default: '\'kubernetes\' \'openshift\''  
```

Comment:
if a comma-separated list, it would be easier to pass this in as:
 * kubernetes,openshift
 * kubernetes
 * openshift